dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/SecretForm.java

```diff
@@ -14,45 +13,28 @@
 public class SecretForm extends Validated {
 
     @NotNull
-    private final String key;
-
-    @NotNull
-    private final String siteId;
-
-    @NotNull
-    private final Map<String, Param> params;
+    private final Map<String, Input> inputParams;
 
     @JsonCreator
-    public SecretForm(@JsonProperty("key") final String key,
-            @JsonProperty("siteId") final String siteId,
-            @JsonProperty("params") final Map<String, Param> params) {
-        this.key = key;
-        this.siteId = siteId;
-        this.params = params;
-    }
-
-    /**
-     * This should contain the unique identifier that differentiates the service
-     * @return
-     */
-    public String getKey() {
-        return key;
+    public SecretForm(final Map<String, Input> inputParams) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)